app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,36 @@
+class TasksController < ApplicationController
+
+  def update_status
+    task_id = task_status_params[:task_id]
+    task_status = task_status_params[:status]
+    task = Task.find(task_id)
+    outcome = Tasks::UpdateTaskStatus.run(employee: current_user, task: task, task_status: task_status)
```

Comment:
I chose the `active-interaction` library to handle business logic related to changing a task's status in order to keep controller actions succinct.